src/test/java/org/apache/commons/io/input/XmlStreamReaderTest.java

```diff
@@ -78,6 +82,26 @@ protected void _testRawNoBomInvalid(final String encoding) throws Exception {
         }
     }
 
+    @Test
+    protected void testNullFileInput() throws IOException {
+        assertThrows(NullPointerException.class, ()-> new XmlStreamReader((File)null));
```

Comment:
Hi @ottobackwards 
The spacing in "() ->" is inconsistent, it should be "() ->" everywhere.